src/views/ipv/page-multiple-doc-check.njk

```diff
@@ -33,7 +33,7 @@
                 divider: 'pages.pageMultipleDocCheck.content.formRadioButtons.separateOptionsInFormText' | translate
               },
               {
-                value: "end",
+                value: "next/claimed-identity",
```

Comment:
Should we do this on a feature flag? This is due to a comment on https://govukverify.atlassian.net/browse/PYIC-2834 where it says ‘How do we make this only available when the F2F is enabled???’